.github/workflows/ci.yaml

```diff
@@ -17,28 +17,33 @@ jobs:
     strategy:
       matrix:
         operating-system: [ ubuntu-latest ]
-        php-version: [ '8.0' ]
+        php-version: [ '8.0', '8.1', '8.2' ]
         coverage-driver: [ pcov, xdebug ]
-        e2e-runner: [ 'bin/infection' ]
+        e2e-runner: [ 'bin/infection', 'build/infection.phar' ]
+        dependencies:
+          - "locked"
+          - "lowest"
+          - "highest"
+        exclude:
```

Comment:
we probably may exclude more cases. Give me nudge if we need to add something else

Reply:
For me it's quite hard to tell at this point, because I don't know what is changed here because we were not testing a wide enough set, or because of the Symfony6 compatibility.

Don't you think it would be best to go towards #1786 first, which allows to test a wider range whilst keeping the Symfony6 compatibility issue out, and then add Symfony6 to fix the compatibility?

If the conflicts are a concern I'm more than happy to take care of them